miwa.rb

```diff
@@ -0,0 +1,45 @@
+class Indicatory
+    puts 'Щосаме вам потрібно порахувати ?'
+    answer = gets.chomp.upcase 
+    if answer == 'Electric'
+      @electric = gets.chomp 
```

Comment:
You need to call product class here with the value of user input. For example `Water.new(value).calculate`